scripts/ltr_msmarco/train_ltr_model.py

```diff
@@ -492,7 +492,6 @@ def save_exp(dirname,
                           max(multiprocessing.cpu_count() // 2, 1))
     #fe.add(RunList('./collections/msmarco-ltr-passage/run.monot5.run_list.whole.trec','t5'))
```

Comment:
Clean up dead code.

Reply:
This is not dead code. So we have two versions of ltr: one as pre filtering stage; one is feeding bert number into ltr. They have two exclusive set of features. The commented out one is for using bert as feature.
Should I just delete or make other comments around those two?

Reply:
yea, comments is fine then.